launchcontainer/launchcontainer.py

```diff
@@ -14,13 +14,34 @@
 from xblock.fields import Scope, String
 from xblock.fragment import Fragment
 
+try:
+    from openedx.core.djangoapps.site_configuration import helpers as siteconfig_helpers
+except ImportError:  # We're not in an openedx environment.
+    siteconfig_helpers = None
 
-log = logging.getLogger(__name__)
 
+logger = logging.getLogger(__name__)
+DEFAULT_WHARF_URL = 'https://wharf.appsembler.com/isc/newdeploy'
```

Comment:
Just FYI: I'm postponing this because I really don't have a good way, yet, to notify the user if the whole URL thing goes bad, and I'm also unsure about our logging situation. This URL could fail on all levels, and it would be hard to determine why. I'll bump this up the priority list, though. 

Many thanks for reviewing this promptly!